src/cpp/core/markdown/sundown/html_blocks.h

```diff
@@ -2,7 +2,7 @@
 /* Command-line: gperf -N find_block_tag -H hash_block_tag -C -c -E --ignore-case html_block_names.txt  */
 /* Computed positions: -k'1-2' */
 
-#if !((' ' == 32) && ('!' == 33) && ('"' == 34) && ('#' == 35) \
+#if (!(' ' == 32) && ('!' == 33) && ('"' == 34) && ('#' == 35) \
```

Comment:
Is this change intentional? It looks odd -- it seems to me the intention really is to negate the whole expression and not just the first piece.

Reply:
@MariaSemple another file we don't own.

Reply:
Looks like it was also accidental, probably part of the same search/replace that changed the other `#if` block